src/material/datepicker/datepicker-input.ts

```diff
@@ -97,59 +69,17 @@ export class MatDatepickerInputEvent<D> {
   },
   exportAs: 'matDatepickerInput',
 })
-export class MatDatepickerInput<D> implements ControlValueAccessor, OnDestroy, Validator {
+export class MatDatepickerInput<D> extends MatDatepickerInputBase<D> {
```

Comment:
Does this need an `inputs` section in the decorator spec? I noticed that the API goldens show some inputs being removed

Reply:
The `inputs` got transferred onto the base class, but since it's not exported through the public API it won't show up in the golden. I think the golden only generates code for the class' own fields.

Reply:
Ok, I assumed tests would be broken if those inputs actually got removed, but just wanted to double check.